glslc/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ add_library(glslc STATIC
 shaderc_default_compile_options(glslc)
 target_include_directories(glslc PUBLIC ${glslang_SOURCE_DIR})
 target_link_libraries(glslc PRIVATE glslang OSDependent OGLCompiler
-  HLSL glslang SPIRV ${CMAKE_THREAD_LIBS_INIT})
+  glslang SPIRV HLSL ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
glslang was already listed before and after HLSL.
I don't know the details but that's usually necessary when there's a circular dependency between the two.   Are you sure that HLSL itself doesn't depend on glslang?

We may require a *second* mention of HLSL, something like:
 
    glslang ... HLSL .. glslang .. HLSL 
?

Reply:
It seems to work as-is for me already, but I suppose I can add the extra link to be on the safe side.